### PR TITLE
add electric conductivity algorithm

### PR DESCRIPTION
**Pull request by sadiqebrahim**
### Describe your change:

added new algorithm for calculating electric conductivity

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: electronics/electric_conductivity.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
+from __future__ import annotations
+
+ELECTRON_CHARGE = 1.6021e-19  # units = C
+5 * 2
```
Comment (CaedenPH) [inline electronics/electric_conductivity.py:4]:
```suggestion
```
?

File: electronics/electric_conductivity.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    ('conductivity', 5.12672e-14)
+    >>> electric_conductivity(conductivity=1000, electron_conc=0, mobility=1200)
+    ('electron_conc', 5.201506356240767e+18)
+    """
+    if (conductivity, electron_conc, mobility).count(0) != 1:
+        raise ValueError("You cannot supply more or less than 2 values")
+    elif conductivity < 0:
+        raise ValueError("Conductivity cannot be negative")
+    elif electron_conc < 0:
+        raise ValueError("Electron concentration cannot be negative ")
```
Comment (CaedenPH) [inline electronics/electric_conductivity.py:31]:
```suggestion
        raise ValueError("Electron concentration cannot be negative")
```

File: electronics/electric_conductivity.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        return (
+            "electron_conc",
+            conductivity / (mobility * ELECTRON_CHARGE),
+        )
+    elif mobility == 0:
+        return (
+            "mobility",
+            conductivity / (electron_conc * ELECTRON_CHARGE),
+        )
+    else:
```
Comment (CaedenPH) [inline electronics/electric_conductivity.py:49]: In what scenario is this else triggered?

File: electronics/electric_conductivity.py [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+def electric_conductivity(
+    conductivity: float,
+    electron_conc: float,
+    mobility: float,
+) -> tuple:
+    """
+    This function can calculate any one of the three -
+    1. Conductivity
+    2, Conductivity
```
Comment (CaedenPH) [inline electronics/electric_conductivity.py:15]: ?

File: electronics/electric_conductivity.py [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+ELECTRON_CHARGE = 1.6021e-19  # units = C
+5 * 2
+
+
+def electric_conductivity(
+    conductivity: float,
+    electron_conc: float,
+    mobility: float,
+) -> tuple:
```
Comment (CaedenPH) [inline electronics/electric_conductivity.py:11]:
```suggestion
) -> tuple[str, float]:
```
A tuple of what?

File: electronics/electric_conductivity.py [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+    """
+    This function can calculate any one of the three -
+    1. Conductivity
+    2, Conductivity
+    3. Electron Mobility
+    given the other two.
```
Comment (CaedenPH) [inline electronics/electric_conductivity.py:17]: This doesn't make sense and could be worded differently
```suggestion
    This is calculated from the other two provided values
```
Or perhaps this is just unnecessary information
```suggestion
```

File: electronics/electric_conductivity.py [hunk 7]
```diff
@@ -0,0 +1,56 @@
+from __future__ import annotations
+
+ELECTRON_CHARGE = 1.6021e-19  # units = C
+5 * 2
+
+
+def electric_conductivity(
+    conductivity: float,
+    electron_conc: float,
```
Comment (CaedenPH) [inline electronics/electric_conductivity.py:9]: What does `conc` stand for?

**Review by cclauss — APPROVED** [review]

**Pull request merged**